src/middlewared/middlewared/plugins/cluster_linux/management.py

```diff
@@ -80,6 +84,63 @@ def __validate_brick_path(self, schema_name, verrors):
                 f'{self.brick_path}: path does not exist on host {self.hostname}.'
             )
 
+        if stat_info is None:
+            return
+
+        if stat_info['type'] != 'DIRECTORY':
+            verrors.add(
+                f'{schema_name}.brick_path',
+                f'{self.brick_path}: path is not a directory on host {self.hostname}.'
+            )
+            return
+
+        if stat_info['is_mountpoint']:
+            verrors.add(
+                f'{schema_name}.brick_path',
+                f'{self.brick_path}: path is mountpoint on host {self.hostname}.'
+            )
+
+        if stat_info['nlink'] != 2:
```

Comment:
You could put this as an `or` operand on line 97 to remove code duplication
```
if stat_info['is_mountpoint'] or stat_info['link'] != 2

Reply:
Derp. I meant to remove that.